src/sql/AliasGenerator.ts

```diff
@@ -0,0 +1,7 @@
+export class AliasGenerator {
+    private aliasCounter: number = 0
+
+    newAlies() {
+        return `table_${++this.aliasCounter}`;
```

Comment:
is multi-threaded access to this class possible?

Reply:
This class is not intended to be used in multiple threads. Only one instance of class is created for each query builder